src/components/GameBoard.jsx

```diff
@@ -2,11 +2,11 @@ import React, { useState, useEffect, Suspense } from "react";
 import SocketEvents from "../lib/enums/socketEvents";
 const ScribbleBoard = React.lazy(() => import("./ScribbleBoard"));
 
-function GameBoard({ initialGameData, initialUserState, socket, userName }) {
+function GameBoard({ initialGameData, initialUserState, socket, userName,users }) {
```

Comment:
is users prop being used here? or just initialuserstate

Reply:
the users variable is being overwritten with const [users, updateUserData] = ....